app/controllers/v1/leaderboards_controller.rb

```diff
@@ -1,5 +1,12 @@
 class V1::LeaderboardsController < ApplicationController
 
+  def index
+    @competition = Competition.find(params[:competition_id])
+    # @leaderboards = current_user.leaderboards(@competition)
+    @ranking = 0
```

Comment:
What is this for?

Reply:
whoops.  i was messing around and forgot to remove it.